### PR TITLE
Rate limit update

### PR DESCRIPTION
**Pull request by RFBomb**
Reduce the sleep timer from 2s to 650ms, in accordance with updated (higher) rate limits allowed as of 7/1. 
Replace all 'sleep()' timers with the new function that uses the calculated rate limit time.
- If implementing this change prior to 7/1, the variable must be changed to 1050ms to stay within the 60/min range in effect until 7/1

### CODE REVIEW

File: src/things/mod.rs [hunk 1]
```diff
@@ -23,10 +23,22 @@ use tracing::instrument;
 use crate::cli::Config;
 use async_trait::async_trait;
 
+// Reddit has a new rate limit as of 7/1/2023:
+// OAuth for authentication: 100 queries per minute per OAuth client id - sleep atleast 0.6s after every call ( 650 ms)
+// not using OAuth for authentication: 10 queries per minute - must sleep atleast 6s between calls ( 6500 ms)
+const SLEEP_TIME:u64 = 650;
+const SLEEP_DUR:Duration = Duration::from_millis(SLEEP_TIME);
+pub async fn prevent_rate_limit() { 
+    println!("\n -- Sleeping for {SLEEP_TIME}ms to prevent rate limiting...\n");
```
Comment (andrewbanchich) [inline src/things/mod.rs:32]: No need for `\n` in `println!` (the `ln` in `println` means it adds line breaks itself). Also could you update this to use `debug!` and remove the `-- `?

**Review by andrewbanchich — CHANGES_REQUESTED** [review]

File: src/things/mod.rs (hunk 1 above)
Comment (RFBomb) [reply src/things/mod.rs]: done.

**Review by andrewbanchich — DISMISSED** [review]

**Comment by andrewbanchich** [comment]
Looks like there's a build failure due to needing to import `debug!`.

**Comment by andrewbanchich** [comment]
Thanks!

**Pull request merged**